app/ts/components/requirements.tsx

```diff
@@ -130,9 +134,34 @@ export const Create = ( { contentHashInput, resolutionAddressInput, loadingInfos
 		areContractsDeployed.value = true
 	}
 	
+	const renewByYear = async () => {
+		const acc = account.peek()
```

Comment:
Rename these to `account` if you accept the previous comment to rename `account` to `maybeAccount`.